Logger/DictToDB.py

```diff
@@ -0,0 +1,63 @@
+import sqlite3
+
+
+class DictToDB:
+
+    __connection = None
+    __cursor = None
+    __tableExists = False
+    __tableName = ""
+
+    connected = False
+
+    def __init__(self, pathToDb, sampleDict, tablename="logdata"):
+        self.__tableName = tablename
+        try:
+            self.__connection = sqlite3.connect(pathToDb)
+        except:
+            self.connected = False
+        self.connected = True
```

Comment:
Be aware that this will reset the `connected` to always return true, even when a exception is thrown!